docs/src/pages/docs/en/internationalization/adding-language-support.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Adding Language Support
+description: Adding Language Support
+layout: ../../../../layouts/MainLayout.astro
+---
+
+Media Chrome supports multiple languages for its UI components. By default, it uses English, but you can include additional languages by loading the corresponding language module.
+
+## Available Languages
+Currently, the supported languages are:
+
+- **English** (default)
+- **Spanish** (`es`)
+- **French** (`fr`)
+- **Portuguese** (`pt`)
+
+## How to Include a Language
+
+### Using HTML
+To add support for a specific language in an HTML file, include the corresponding language module using a `<script>` tag:
+
+```html
+<script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@4.8.0/dist/lang/pt.js/+esm"></script>
```

Comment:
```suggestion
<script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@4.8.0/dist/lang/pt.js"></script>
```

we have to use the non /+esm endpoint for this to work.

also here we should mention that for this to work they have to use the non-bundled media-chrome dist
this one https://cdn.jsdelivr.net/npm/media-chrome@4
instead of https://cdn.jsdelivr.net/npm/media-chrome@4/+esm

this is needed because we need to set state in the `i18n.js` module which only works if it can be imported separately.